src/test/test-string-util.c

```diff
@@ -314,7 +314,7 @@ static void test_strshorten(void) {
         assert_se(strlen(strshorten(s, 6)) == 6);
         assert_se(strlen(strshorten(s, 12)) == 6);
         assert_se(strlen(strshorten(s, 2)) == 2);
-        assert_se(strlen(strshorten(s, 0)) == 0);
+        assert_se(isempty(strshorten(s, 0)));
```

Comment:
Please drop this, this is testing code for `strshorten` and we don't want to mix `isempty` into it.